enhancements/installer/connected-onprem-assisted-installer.md

```diff
@@ -0,0 +1,236 @@
+---
+title: assisted-installer-on-prem
+authors:
+  - "@mhrivnak"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-07-16
+last-updated: 2020-07-16
+status: implementable
+see-also:
+  - "/enhancements/installer/connected-assisted-installer.md"  
+---
+
+# Assisted Installer On-Prem
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The assisted installer, [previously proposed and
+accepted](connected-assisted-installer.md), includes a service running on
+cloud.redhat.com to which agents connect and wait for orders on how to
+A) provision the host they are running on, and B) join or form a cluster.
+
+This enhancement proposes to enable a user to run that "assistant" service
+on-premise, inside their own network and on their own infrastructure, for
+one-time-use to create a new cluster
+
+## Motivation
+
+Many users will not want a service that can provision hosts inside their
+network to be run by a third party outside of their network. Reasons can
+include trust, control, and reproducibility. Thus those users will prefer to
+run the service on-premise.
+
+### Goals
+
+* Users can run the entire assisted install workflow with software that runs inside their network.
+
+### Non-Goals
+
+* Disconnected and air-gapped environments are not included in this enhancement.
+* Adding Nodes on day 2 will be addressed separately.
+* Utilizing a local container registry for mirrored content will be addressed in a future enhancement.
+
+## Proposal
+
+A user will visit cloud.redhat.com to download a live ISO with which they can
```

Comment:
So this is not intended to ever be something that they would "install", just something they run some number of times and throw away?

Reply:
yes, ideally in a follow up step the bootstrap and assisted share the same node and later on when you pivot you would displace them both.